api/app/clients/GoogleClient.js

```
@@ -638,185 +597,230 @@ class GoogleClient extends BaseClient {
       }
 
-      if (this.options?.promptPrefix?.length) {
-        requestOptions.systemInstruction = {
-          parts: [
-            {
-              text: promptPrefix,
-            },
-          ],
-        };
+      const { instances } = _payload;
+      const { messages: messages, context } = instances?.[0] ?? {};
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)